src/auro-menu.js

```diff
@@ -39,31 +41,59 @@ class AuroMenu extends LitElement {
 
   firstUpdated() {
     this.addEventListener('keydown', this.handleKeyDown);
-    this.addEventListener('mousedown', this.handleClick);
-    this.addEventListener('click', this.selectedOption);
+    this.addEventListener('click', this.makeSelection);
```

Comment:
Using `click` only checks the option when the mouse click is up. This was discussed with the last iteration. 

```suggestion
    this.addEventListener('mousedown', this.makeSelection);
```